contentful/apps
# [INTEG-976] Add ESLint to AICG and fix errors

## Purpose

We did not have ESLint configured for the AI Content Generator app. This PR adds a standard configuration, and fixes the errors and warnings in the app.

## Approach

Installed ESLint as a dev dependency and and ran `npm init @eslint/config` to get a basic configuration set up using plugins for react, react hooks, and typescript. 

I temporarily added some files to the `.eslintignore`, namely the V1 components and test files, along with TODOs to clean those up later.

## Testing steps

Test the basic actions in the app to ensure that it all behaves as expected. Run `npm run lint` and you shouldn't see any warnings or errors.
